### ISSUE TITLE
Are bookmarks a pdf accessibility feature?

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#13]
I have a strong suspicion that the "bookmark" value for accessibility feature was intended for PDFs, which is more of a table of contents. There's a [WCAG technique](https://www.w3.org/WAI/WCAG21/Techniques/pdf/PDF2.html) about adding them, after all, and I don't recall promoting this for ebooks when we created the vocabulary.

Putting aside the fact we don't have a common means of distributing the kind of digital "dog-ear" bookmarks reading systems provide, is there a special accessibility case for them even if we could? They can be generally useful for all users, but what would be the scenario where they add accessibility that is missing? I can't imagine professors distribute dog-eared copies of print course material to their students, for example.

Anyway, if this is a PDF feature, it would be good to clearly label it as such and equate it with a table of contents.
